src/artemis/experiment_plans/fast_grid_scan_plan.py

```diff
@@ -254,10 +233,7 @@ def run_gridscan_and_move(
     # once we have the results, go to the appropriate position
     artemis.log.LOGGER.info("Moving to centre of mass.")
     with TRACER.start_span("move_to_result"):
-        yield from move_xyz(
-            fgs_composite.sample_motors,
-            xray_centre,
-        )
+        yield from move_x_y_z(fgs_composite.sample_motors, *xray_centre)
```

Comment:
Must: The previous function waited, this now doesn't I think we probably want to still

Reply:
I still think we probably want to do this